echo.js

```diff
@@ -1,7 +1,11 @@
 function echo(){
-  // ...
+  for (var i = 0; i < arguments.length; i++) {
+  	console.log(arguments[i]) + "<br>";
```

Comment:
> 'console' is not defined.

This warning is a fluke – will try and get it sorted out for the next assignment. Sorry for any confusion!